posts/views.py

```diff
@@ -1,3 +1,15 @@
-from django.shortcuts import render
+from django.shortcuts import redirect
+from posts.models import Post
+from django.views import generic
 
-# Create your views here.
+
+def PostCreateView(request):
+    if (len(request.POST['content']) <= 256):
+        post = Post(content=request.POST['content'], author=request.user)
+        post.save()
+    return redirect(request.META['HTTP_REFERER'] or 'landingpage')
```

Comment:
There is nothing that can go wrong when using the app normally. The only error would be that the content was longer than 256 chars, but as the form disallows this content length, the error can only occur when creating a custom POST request, which is not intended.